docs/developers/developers-setup.md

```diff
@@ -168,7 +168,7 @@ meson test
 To install Diffuse on your system (e.g. `/usr/local/`):
```

Comment:
I'm actually wondering, as I can't test on macOS, is the app installed in `/usr/local/`? (that's the case for Linux in most distributions, but I'm unsure for macOS)

Reply:
```sh
~ which diffuse
/opt/homebrew/bin/diffuse
```

And there is a shortcut in the list of applications, but I'm not sure where it lives in the file system.

Reply:
Could you change `/usr/local/` with `/opt/homebrew/`? So it's done as well?

Reply:
I believe this depends on the architecture, arm64 vs. Intel. We can mention both.

Reply:
wait... isn't `/opt/homebrew/` the directory if you installed Diffuse using `brew`? I don't know much about Mac, but I would find it weird that `meson install` would actually install it in a brew-specific directory, don't you think?

Reply:
Yep, `meson install` will install Diffuse into /Applications and to /usr/local/bin, by default, in Mac. 

I wrote https://github.com/MightyCreak/diffuse/pull/165#discussion_r1005489000 about that as well. 

Maybe you would you like to fix https://github.com/MightyCreak/diffuse/pull/165#discussion_r1005489000 and https://github.com/MightyCreak/diffuse/pull/165#discussion_r1005739383 in this PR?

Reply:
Right, `/Applications` worked without `sudo` because I'm a member of the "admin" group on my system.

I'll do another pass here.

Regarding `/usr/local` vs. `/opt/homebrew`, I'm seeing this, with fairly default settings:

```
➜  build git:(b-macos-next-diff) meson install
ninja: Entering directory `/Users/kirill/git/diffuse/build'
ninja: no work to do.
Installing subdir /Users/kirill/git/diffuse/data/usr/share to /opt/homebrew/share
Installing /Users/kirill/git/diffuse/data/usr/share/omf/diffuse/diffuse-C.omf to /opt/homebrew/share/omf/diffuse
Installing /Users/kirill/git/diffuse/data/usr/share/omf/diffuse/diffuse-ru.omf to /opt/homebrew/share/omf/diffuse
Installing /Users/kirill/git/diffuse/data/usr/share/omf/diffuse/diffuse-it.omf to /opt/homebrew/share/omf/diffuse
Installing /Users/kirill/git/diffuse/data/usr/share/omf/diffuse/diffuse-cs.omf to /opt/homebrew/share/omf/diffuse
Installing /Users/kirill/git/diffuse/data/usr/share/man/it/man1/diffuse.1 to /opt/homebrew/share/man/it/man1
Installing /Users/kirill/git/diffuse/data/usr/share/man/cs/man1/diffuse.1 to /opt/homebrew/share/man/cs/man1
Installing /Users/kirill/git/diffuse/data/usr/share/man/ru/man1/diffuse.1 to /opt/homebrew/share/man/ru/man1
Installing /Users/kirill/git/diffuse/data/usr/share/man/man1/diffuse.1 to /opt/homebrew/share/man/man1
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/csharp.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/vb.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/bash.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/makefile.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/verilog.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/python.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/javascript.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/patch.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/octave.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/css.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/ini.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/pascal.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/jsp.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/html.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/c++.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/php.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/sql.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/xml.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/fortran.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/csh.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/tcl.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/gettext.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/objective-c++.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/rust.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/opencl.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/json.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/ruby.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/r.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/java.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/glsl.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/vhdl.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/perl.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/diffuse/syntax/erlang.syntax to /opt/homebrew/share/diffuse/syntax
Installing /Users/kirill/git/diffuse/data/usr/share/gnome/help/diffuse/it/diffuse.xml to /opt/homebrew/share/gnome/help/diffuse/it
Installing /Users/kirill/git/diffuse/data/usr/share/gnome/help/diffuse/cs/diffuse.xml to /opt/homebrew/share/gnome/help/diffuse/cs
Installing /Users/kirill/git/diffuse/data/usr/share/gnome/help/diffuse/ru/diffuse.xml to /opt/homebrew/share/gnome/help/diffuse/ru
Installing /Users/kirill/git/diffuse/data/usr/share/gnome/help/diffuse/C/diffuse.xml to /opt/homebrew/share/gnome/help/diffuse/C
Installing subdir /Users/kirill/git/diffuse/src/diffuse/vcs to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/vcs_registry.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/git.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/svk.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/rcs.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/svn.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/bzr.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/mtn.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/__init__.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/folder_set.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/darcs.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/cvs.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/hg.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing /Users/kirill/git/diffuse/src/diffuse/vcs/vcs_interface.py to /opt/homebrew/share/diffuse/diffuse/vcs
Installing po/cs/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/cs/LC_MESSAGES
Installing po/de/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/de/LC_MESSAGES
Installing po/es/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/es/LC_MESSAGES
Installing po/fr/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/fr/LC_MESSAGES
Installing po/it/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/it/LC_MESSAGES
Installing po/ja/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/ja/LC_MESSAGES
Installing po/ko/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/ko/LC_MESSAGES
Installing po/pl/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/pl/LC_MESSAGES
Installing po/pt/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/pt/LC_MESSAGES
Installing po/pt_BR/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/pt_BR/LC_MESSAGES
Installing po/ru/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/ru/LC_MESSAGES
Installing po/sv/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/sv/LC_MESSAGES
Installing po/th/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/th/LC_MESSAGES
Installing po/zh_CN/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/zh_CN/LC_MESSAGES
Installing po/zh_TW/LC_MESSAGES/diffuse.mo to /opt/homebrew/share/locale/zh_TW/LC_MESSAGES
Installing /Users/kirill/git/diffuse/build/data/diffuserc to /opt/homebrew/etc
Installing /Users/kirill/git/diffuse/data/icons/hicolor/scalable/apps/io.github.mightycreak.Diffuse.svg to /opt/homebrew/share/icons/hicolor/scalable/apps
Installing /Users/kirill/git/diffuse/data/icons/hicolor/symbolic/apps/io.github.mightycreak.Diffuse-symbolic.svg to /opt/homebrew/share/icons/hicolor/symbolic/apps
Installing /Users/kirill/git/diffuse/data/icons/diffuse.icns to /Applications/Diffuse.app/Contents/Resources
Installing /Users/kirill/git/diffuse/build/src/diffuse/diffuse_impl to /opt/homebrew/bin
Installing /Users/kirill/git/diffuse/build/src/diffuse/diffuse to /opt/homebrew/bin
Installing /Users/kirill/git/diffuse/build/src/diffuse/mac_launcher.sh to /Applications/Diffuse.app/Contents/MacOS
Installing /Users/kirill/git/diffuse/build/src/diffuse/Info.plist to /Applications/Diffuse.app/Contents
Installing /Users/kirill/git/diffuse/src/diffuse/__init__.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/constants.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/dialogs.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/main.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/preferences.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/resources.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/utils.py to /opt/homebrew/share/diffuse/diffuse
Installing /Users/kirill/git/diffuse/src/diffuse/widgets.py to /opt/homebrew/share/diffuse/diffuse
```

Reply:
Do you have /opt/homebrew in any if your env variables?
```
export | grep homebrew
```

Reply:
```
HOMEBREW_CELLAR=/opt/homebrew/Cellar
HOMEBREW_PREFIX=/opt/homebrew
HOMEBREW_REPOSITORY=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
MANPATH=/opt/homebrew/share/man::
PATH=/Users/kirill/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

Reply:
Ok. Yeah, I don't have the env variable `HOMEBREW_PREFIX` set and if I ask brew for its config I get `HOMEBREW_PREFIX: /usr/local`.

```
$ brew config
HOMEBREW_VERSION: 3.6.4
ORIGIN: https://github.com/Homebrew/brew
HEAD: 99682c5af3b48e7b37bf9c565781800153ee88fb
Last commit: 4 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 20a991bff175a2e7e89faf9fd9f4456163bce42f
Core tap last commit: 3 weeks ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
...
```

I found this hint in the install script of brew:
https://github.com/Homebrew/install/blob/master/install.sh#L132

Reply:
I'm just a user here, set it up with the install script and didn't touch anything. I believe it's fair to assume that other users will see the same behavior. No idea if it's correct though.

Reply:
Yep yep, https://github.com/Homebrew/install/blob/master/install.sh#L132Y indicates that users of M1 CPUs will get this path?